### ISSUE TITLE
Update EFS limits

### ISSUE THREAD

**Issue by yevgen87** [jantman/awslimitchecker#358]
### Current EFS limits
https://docs.aws.amazon.com/efs/latest/ug//limits.html#limits-efs-resources-per-account-per-region

### version
awslimitchecker 5.0.0

### The required changes:

```diff
$ git diff 3ba39a5cf73ee61e3ca58f65885ff66066f126d4 294e62a1352fd015519874814d67ef53a1c7fdfa
diff --git a/awslimitchecker/services/efs.py b/awslimitchecker/services/efs.py
index 0250659..f2bcfb3 100644
--- a/awslimitchecker/services/efs.py
+++ b/awslimitchecker/services/efs.py
@@ -98,10 +98,12 @@ class _EfsService(_AwsService):
         if self.limits != {}:
             return self.limits
         limits = {}
+        self.connect()
+        region_name = self.conn._client_config.region_name
         limits['File systems'] = AwsLimit(
             'File systems',
             self,
-            10,
+            70 if region_name == 'us-east-1' else 125,
             self.warning_threshold,
             self.critical_threshold,
             limit_type='AWS::EFS::FileSystem',
```

**Comment by jantman** [comment]
This has been fixed in #360 and will be released shortly.

**Comment by jantman** [comment]
This has been released in [5.1.0](https://github.com/jantman/awslimitchecker/releases/tag/5.1.0) and is now live on PyPI. Thanks so much!
